blocks/columns/columns.css

```diff
@@ -19,15 +19,62 @@
   display: block;
 }
 
-@media (min-width: 900px) {
+@media (min-width: 1023px) {
   .columns > div {
     align-items: center;
     flex-direction: unset;
-    gap: 32px;
   }
 
   .columns > div > div {
     flex: 1;
     order: unset;
   }
+
+  .pod > div > div:not(.columns-img-col) * {
+    padding-left: 70px;
+    padding-right: 70px;
+    max-width: 405px;
+  }
+}
+
+.pod h2 {
+  font-size: 1.75rem;
+}
+
+.pod p {
+  font-size: 1.125rem;
+  line-height: 1.5;
+}
+
+@media (max-width: 1023px) {
```

Comment:
As mobile should be the default behaviour all this logic should probably be outside a media query

Reply:
Good point, I moved the logic outside of the media query